app/src/test/java/com/google/android/ground/model/submission/NumberTaskDataTest.kt

```diff
@@ -0,0 +1,48 @@
+package com.google.android.ground.model.submission
+
+import junit.framework.TestCase.assertEquals
+import junit.framework.TestCase.assertFalse
+import junit.framework.TestCase.assertNotNull
+import junit.framework.TestCase.assertNull
+import junit.framework.TestCase.assertTrue
+import org.junit.Test
+
+class NumberTaskDataTest {
```

Comment:
Let's remove the tests from here and add any missing coverage to the task fragment test